.github/workflows/ci.yml

```diff
@@ -54,27 +83,14 @@ jobs:
       - name: Install platform dependencies
         run: |
           sudo apt -y update
-          sudo apt -y install libcurl4-openssl-dev libssl-dev pkg-config
+          sudo apt -y install libcurl4-openssl-dev libssl-dev pkg-config libxml2-dev libxslt-dev
       - uses: actions/setup-python@v4
         with:
           python-version-file: '.python-version'
-          cache: 'pip'
-          cache-dependency-path: |
-            requirements.txt
-            requirements/*.txt
-      - name: Cache common Python cache paths
-        uses: actions/cache@v3
+      - name: Restore built Python environment from deps
+        uses: actions/cache/restore@v3
         with:
-          path: |
-            .cache
-            .mypy_cache
```

Comment:
Question: are we intentionally removing the mypy cache?

Reply:
nope, I just goofed! will restore.